packages/mobile/src/components/divider/divider.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { Text, View } from '~/components/components';
+import { useCustomTheme } from '~/hooks/hooks';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+type Props = {
+  text?: string;
+};
+
+const Divider: FC<Props> = ({ text }) => {
+  const { colors } = useCustomTheme();
+
+  return (
+    <View
+      style={[
+        styles.container,
+        globalStyles.flexDirectionRow,
+        globalStyles.alignItemsCenter,
+      ]}
+    >
+      <View
+        style={[
+          text ? styles.halfLine : styles.line,
+          { backgroundColor: colors.line },
+        ]}
+      />
+      {text && (
```

Comment:
This will help to avoid empty string error
```suggestion
      {!!text && (
```

Reply:
corrected